docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
     image: postgres:latest
     profiles:
       - context_storage
+      - stats
```

Comment:
Why not store the data in two distinct databases or two distinct tables? I think it's more feasible than creating multiple postgres instances

Reply:
I introduced a new variable, but creating two psql databases on docker image startup is non-trivial, so right now both Superset and context storage use the 'test' database. This does not seem to cause any issues, since they use different tables

Reply:
Created a separate container